CHANGELOG.md

```diff
@@ -4,12 +4,9 @@
 
 [Full Changelog](https://github.com/ably/ably-java/compare/v1.2.37...v1.2.38)
 
-**Breaking changes:**
```

Comment:
I think we shouldn't put this fix in the "breaking changes" section, because we fixed this breaking change and treated it as a bug